.github/workflows/rust.yml

```diff
@@ -0,0 +1,25 @@
+name: Rust
```

Comment:
I would say let's rename this file to `ci.yml` and name of this workflow to `CI`.